spring-cloud-kubernetes-dependencies/pom.xml

```diff
@@ -44,36 +42,12 @@
 				<version>${project.version}</version>
 			</dependency>
 
-			<dependency>
-				<groupId>org.springframework.cloud</groupId>
-				<artifactId>spring-cloud-kubernetes-archaius</artifactId>
```

Comment:
Add modules back and comment them out.

Reply:
This is already down there commented out in line 51, I've moved all the commented modules together. Is there anything else needed regarding this one?